bitbots_bringup/scripts/check_robot.py

```diff
@@ -0,0 +1,285 @@
+        both_okay = both_okay and check_pressure(right_pressure, -1, 1, "right")
+        if both_okay:
+            print("Pressures correct after calling zero service. You can continue normally.\n")
+        else:
+            print_warn("Pressures still wrong. Please investigate the problem.\n")
+
+    # check pressure values when robot in walkready
+    input("Please put the robot standing on the ground and press enter")
+    both_okay = True
+    both_okay = both_okay and check_pressure(left_pressure, 20, 40, "left")
+    both_okay = both_okay and check_pressure(right_pressure, 20, 40, "right")
+    if both_okay:
+        print("Pressure seems to be okay\n")
+    else:
+        print_warn("Problem with pressure. "
+                   "Please recalibrate the sensors using rosrun bitbots_ros_control pressure_calibaration\n")
+
+    # todo check pickup hcm state
+
+    # check fall front
+    input("\nNext we will check front falling detection.\n "
+          "Please let the robot fall on its belly, but hold it to prevent damage. "
+          "The robot should perform a safety motion. Afterwards it should stand up by itself. "
+          "Press enter when you're done.\n")
+
+    input("\nNext we will check back falling detection.\n "
+          "Please let the robot fall on its back, but hold it to prevent damage. "
+          "The robot should perform a safety motion. Afterwards it should stand up by itself. "
+          "Press enter when you're done.\n")
+
+    # check walk motion
+    walk_pub = rospy.Publisher("/cmd_vel", Twist, queue_size=1)
+    text = input(
+        "\nWe will check walking of the robot. After pressing enter, robot will start walking in different directions. "
+        "It will stop when it is finished. Please make sure there is space and catch it if it falls. Press y if you want to check walking.")
+    if text == "y":
+        cmd_msg = Twist()
+        cmd_msg.linear.x = 0.1
+        walk_pub.publish(cmd_msg)
+        rospy.sleep(5)
+        cmd_msg.linear.x = -0.1
+        walk_pub.publish(cmd_msg)
+        rospy.sleep(5)
+        cmd_msg.linear.x = 0.0
+        cmd_msg.linear.y = 0.1
+        walk_pub.publish(cmd_msg)
+        rospy.sleep(5)
+        cmd_msg.linear.y = -0.1
+        walk_pub.publish(cmd_msg)
+        rospy.sleep(5)
+        cmd_msg.linear.y = 0
+        walk_pub.publish(cmd_msg)
+        rospy.sleep(1)
+
+    # check kick motion
+    text = input("\nWe will check the kick motion. Please hold make sure the robot is safe. "
+                 "Press y if you want to perform this test.")
+    client = actionlib.SimpleActionClient('dynamic_kick', KickAction)
+    goal = KickGoal()
+    goal.header.stamp = rospy.Time.now()
+    goal.header.frame_id = "base_footprint"
+    goal.ball_position.x = 0.2
+    goal.ball_position.y = -0.09
+    goal.ball_position.z = 0
+    goal.kick_direction = Quaternion(*quaternion_from_euler(0, 0, 0))
+    goal.kick_speed = 1
+
+
```

Comment:
This formatting looks confusing, can you put the functions at the top to the others?